Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -1233,6 +1265,31 @@ public protocol PurchasesSwiftType: AnyObject {
     func eligibleWinBackOffers(
         forPackage package: Package
     ) async throws -> [WinBackOffer]
+
+    /**
+     * Fetches the virtual currencies for the current subscriber.
+     *
+     * - Returns: The ``VirtualCurrencies`` object containing the virtual currencies for the subscriber.
+     *
+     * #### Related Articles
+     * -  [Virtual Currencies](https://www.revenuecat.com/docs/offerings/virtual-currency)
+     */
+    func virtualCurrencies() async throws -> VirtualCurrencies
+
+    /**
+     * Fetches the virtual currencies for the current subscriber.
+     *
+     * - Parameter forceRefresh: Whether or not the SDK must refresh virtual currencies from the backend,
+     * bypassing the SDK's cache. Pass in `true` to force the SDK to fetch the most recent value
+     * from the RevenueCat backend.
+     * - Returns: The ``VirtualCurrencies`` object containing the virtual currencies for the subscriber.
+     *
+     * #### Related Articles
+     * -  [Virtual Currencies](https://www.revenuecat.com/docs/offerings/virtual-currency)
+     */
+    func virtualCurrencies(
+        forceRefresh: Bool
+    ) async throws -> VirtualCurrencies
```

Comment:
same here, any reason to have them as separate functions?

Reply:
Consolidated these in f7433107758dfffa31da562cd7739b9d5c569040!